tools/spatialDraw/index.html

```diff
@@ -49,12 +49,14 @@
             background: rgba(0,0,0,0.9);
             border-radius: 0 20px 20px 0;
             width: fit-content;
-            margin: 7.5% 0;
             padding: 10px;
             display: flex;
             flex-direction: column;
             justify-content: space-evenly;
             align-items: center;
+            left: 0;
+            top: calc(50% - 171px); /* 342px height + padding in inspector */
```

Comment:
If you want to make this future-proof you can do:
```css
top: 50%;
transform: translateY(-50%);
```
where the transform's -50% is calculated against the element's height for some reason